addpage.php

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html lang='en' font-family = 'Segoe UI, Helvetica, Verdana'>
+<head>
+    <title>collection_app_ahardman</title>
+    <link href='normalize.css' rel='stylesheet' type='text/css'>
```

Comment:
i ws told from mike that normalize shouldn't be first as it can cause some issues with priority and overwrite some of your CSS.